distributed/worker.py

```diff
@@ -1960,6 +1936,12 @@ def select_keys_for_gather(self, worker, dep):
         return deps, total_bytes
 
     async def gather_dep(self, worker, dep, deps, total_nbytes, cause=None):
+        """
+        worker: str
+        dep: TaskState object
+        deps: list of keys of dependencies to gather
```

Comment:
I support docstrings generally, but when we add them we should probably adhere to numpdoc standard.  

I know that you know this stuff better than 99.999999% of the population (probably an underestimate) but I'll do what I always do, and copy in this doc: https://docs.dask.org/en/latest/develop.html#docstrings

Reply:
Yep, these were notes to myself that I forgot to remove, but I'll make it a proper docstring instead (notes for my future self)